middlewares/validators/user.js

```diff
@@ -19,8 +18,7 @@ const updateUser = async (req, res, next) => {
     instagram_id: joi.string().optional(),
     site: joi.string().optional(),
     isMember: joi.boolean().optional(),
-    tokens: joi.object().optional(),
-    website: joi.string().optional()
```

Comment:
I just realised we're sending `website` from the signup form, I'll change the API contract, please remove `site` and keep `website`

Reply:
@TD-17

Reply:
Let's remove `company_name`

Reply:
yeah okay